package/gluon-web-model/javascript/gluon-web-model.js

```diff
@@ -517,7 +517,7 @@
 		for (var i = 0, node; (node = nodes[i]) !== undefined; i++) {
 			var events = node.getAttribute('data-update').split(' ');
 			for (var j = 0, event; (event = events[j]) !== undefined; j++) {
-				bind(node, event, update);
+				bind(node, event, function () {setTimeout(update, 0);});
```

Comment:
Can you describe what happens here? This sounds like it introduces a delay in the update callback, like something here is racy.

Reply:
I think this is fine - the "update" function should be idempotent, so as long as we have a call after each update trigger, calling it "later" (at the next opportunity - basically still immediately, with certain caveats) shouldn't be a problem.

Reply:
`setTimeout` with time 0 basically puts the callback immediately at the end of the event queue, so the browser has time to do any other events and handlers. Since the form's inputs are updated with the reset data _after_ the `onreset` handlers on the form, we give the browser time to do that before we can process the new values.

Reply:
That was my interpretation of the change as well, thanks for the confirmation.